src/schema/sale_artwork.js

```diff
@@ -272,6 +261,42 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLFloat,
         deprecationReason: "Favor `minimum_next_bid`",
       },
+      my_increments: {
+        type: new GraphQLList(BidIncrementsFormatted),
+        resolve: async (
+          { _id, minimum_next_bid_cents, sale_id, symbol },
+          options,
+          request,
+          { rootValue: { incrementsLoader, lotStandingLoader, saleLoader } }
+        ) => {
+          const myLotStandings = await lotStandingLoader({
+            sale_artwork_id: _id,
+          })
+
+          const myCurrentMax = get(
+            myLotStandings,
+            "0.max_position.max_bid_amount_cents"
+          )
+
+          const myMinimumNextBid = myCurrentMax
+            ? maxBy([myCurrentMax, minimum_next_bid_cents])
+            : minimum_next_bid_cents
+
+          return bid_increments_calculator({
+            sale_id,
+            saleLoader,
+            incrementsLoader,
+            minimum_next_bid_cents: myMinimumNextBid,
+          }).then(bid_increments => {
+            return bid_increments.map(increment => {
+              return {
+                cents: increment,
+                display: formatMoney(increment / 100, { symbol, precision: 0 }),
```

Comment:
This `/ 100` feels like it will break when we do a JPY auction, is there a `subunit_to_unit` (or the equivalent) value around?

Reply:
There is [this JSON](https://github.com/artsy/metaphysics/blob/e882ea9a3e447d2afc2b13c40198e0f4af55207d/src/lib/currency_codes.json) file where you can look up a currency code and get that value if you think it's a good idea to switch to that.

Reply:
Haha, this is what I get for copy-pasta-ing the code from above 😆